pkg/components/velero/component_test.go

```diff
@@ -210,8 +210,8 @@ component "velero" {
     }
```

Comment:
Typo in commit message `Toletration to Toleration`. :-)

Reply:
yes, thats what the commit does
change the typo from Toletration to Toleration.

Reply:
Yes, but the spelling is still wrong in the commit message, that's what I meant. You are still using `Toletration to Toletration`. It should be `Toletration to Toleration`